<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Album photo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
	background: #eee;
}

QFrame {
	border: none;
}

QPushButton {
	background: #fff;
	height: 100%;
	border-radius: 7px;
	border: 2px solid white;
	color : black;
	font: 400 13pt &quot;Segoe UI&quot;;
}

QPushButton:hover {
	background: rgba(255, 255, 255, 0.3);
}

QPushButton:focus {
	outline: none; 
	border: 2px solid #5BFF6D;
}


QLineEdit {
    background-color: #FFF;
    border: 2px solid #B0BEC5;
    border-radius: 5px;
    padding: 5px; 
    selection-background-color: #AED6F1;
}

QLineEdit:focus {
    border: 2px solid #ccc; 
}

QLineEdit::placeholder {
    color: #B0BEC5; 
}

QLineEdit::hover {
    border: 2px solid #B0BEC5;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="90,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background: #eee;
}

#albums {
	icon: url(:icon/albums.png);
	text-align: center;
}

#files {
	icon: url(:icon/files.png);
	text-align: center;
	padding-left: 2px;
}

#newAlbum {
	icon: url(:icon/newAlbum.png);
	text-align: center;
	padding-left: 1px;
}

#quit {
	icon: url(:icon/quit.png);
	text-align: center;
	padding-left: 4px;
}

#albumPhoto {
	icon: url(:icon/albumPhoto.png);
	text-align: center;
	padding-left: 1px;
}

#albums:hover {
	background: rgba(255, 255, 255, 0.3);
}

QPushButton:hover {
	background: rgba(255, 255, 255, 0.3);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="albums">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>38</width>
           <height>38</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="files">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newAlbum">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="albumPhoto">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAlbum">
    <property name="title">
     <string>Select window</string>
    </property>
    <addaction name="actionAlbum"/>
    <addaction name="actionPhotos"/>
    <addaction name="actionFiles"/>
    <addaction name="actionPhoto_editor"/>
   </widget>
   <addaction name="menuAlbum"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAlbum">
   <property name="text">
    <string>Album</string>
   </property>
  </action>
  <action name="actionPhotos">
   <property name="text">
    <string>Photos</string>
   </property>
  </action>
  <action name="actionFiles">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="actionPhoto_editor">
   <property name="text">
    <string>Photo editor</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="text">
    <string>Database</string>
   </property>
  </action>
  <action name="actionCreer_Album">
   <property name="text">
    <string>Creer Un Album</string>
   </property>
   <property name="toolTip">
    <string>Creer Album</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>albums</tabstop>
  <tabstop>files</tabstop>
  <tabstop>newAlbum</tabstop>
  <tabstop>albumPhoto</tabstop>
  <tabstop>quit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
